ext/graphql_libgraphqlparser_ext/visitor_functions.c

```diff
@@ -227,6 +227,16 @@ void named_type_end_visit(const struct GraphQLAstNamedType* node, void* builder_
   end_visit(builder_ptr);
 }
 
+int null_value_begin_visit(const struct GraphQLAstNullValue* node, void* builder_ptr) {
+  VALUE rb_node = build_rb_node((struct GraphQLAstNode*) node, "NullValue", builder_ptr);
+  VALUE rb_string = rb_str_new2("null");
```

Comment:
That would be nice, here's the corresponding code in the Ruby parser: 

https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/language/parser.y#L191